cmd/installer/root.go

```diff
@@ -50,9 +72,24 @@ func AddPack(packPath string, checkEula, extractEula bool) error {
 	ui.Extract = extractEula
 
 	if err = pack.install(Installation, checkEula || extractEula); err != nil {
+		if dropPreInstalled {
+			log.Error("Error installing pack, reverting temporary pack to original state")
+			if err := utils.MoveFile(backupPackPath, fullPackPath); err != nil {
```

Comment:
I know this is nit-picking, but there could be a scenario where the user might hit ctrl+c during pack decompression and that will leave a dirty folder `Vendor/PackToReinstall/1.2.3`, so when you try to revert the `_tmp` folder to its original form it will return an error saying that the file/directory already exists

Reply:
This conversations was not resolved? Does this mean the feedback was not considered but the code merged anyhow?

Reply:
Hi Joachim, we talked offline then I merged it right after. The fix was done on [line 72](https://github.com/Open-CMSIS-Pack/cpackget/pull/78/files/212ab074484be458bb3ea314feb3b45eb689661e..c954da055a51c646f466762a5bd731f3db2144f9#diff-d8e18e1da938b2bbf8317652471fa3ead97081225a4c3ccd0aa6422523d99c14R72) . I let Luis know to mark conversations as resolved next time this happens.